src/interfaces/operation.ts

```diff
@@ -98,74 +98,99 @@ export interface Operation {
 }
 
 /**
+ * Group of actions.
+ *
+ * It shows name of a group, drop down list of actions
+ * and some actions which are shown in case list is covered.
  * Группа действий, показывает название группы и раскрываемые список ее действий,
  * а также несколько действий рядом с группой, которые видны не раскрывая список.
- *
- * @param text - отображаемое название группы действий
- * @param actions - список действий в группе
- * @param maxGroupVisualButtonsCount - сколько действий будут видны, не раскрывая список
  */
 export interface OperationGroup {
+    /**
+     * ???
```

Comment:
It's meant as unique identifier for the operation group; it is used by [widget.options.actionGroups](https://github.com/tesler-platform/tesler-ui/blob/b3f517597e6d52ecffa49524f7632700244edabd/src/interfaces/widget.ts#L237) and [useWidgetOperation](https://github.com/tesler-platform/tesler-ui/blob/master/src/hooks/useWidgetOperations.ts) to decide which group of operations to include or exclude, as operations groups configured in business component service and widgets sharing the same BC might need a different set of available actions/action groups